### PR TITLE
Update

### PR DESCRIPTION
**Pull request by adlenafane**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -66,15 +66,14 @@ <h1 class='score'>High scores</h1>
...
         render: function () {
-          console.log(score);
           if (newScores[scoreIndex] <= scores[scores.length - 1].score) {
             this.score = newScores[scoreIndex];
             scores.push({ name: this.name, score: this.score });
           } else {
             // check the score is a valid score
             var checkScore = this.score;
             var isValid = newScores.some(function (validScore) { return checkScore == validScore; });
-            console.log(score, scoreIndex, newScores[scoreIndex]);
```
Comment (mgesbert) [inline index.html]: :clap:

File: README.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
 # Join the team
 
-Get your name in top of the list with the highest score on https://alkemics.github.io/sf2/ and send us a mail on steve@alkemics.com with the URL.
+Our developers are the best at Street Fighter, here is our high score page:
```
Comment (mgesbert) [inline README.md]: either pluralize score here, or singularize it in the next sentence

**Comment by mgesbert** [comment]
1 comment, otherwise LGTM

**Comment by sebastiendavid** [comment]
lgtm

File: README.md (hunk 2 above)
Comment (adlenafane) [inline README.md]: @mgesbert done thx

**Pull request merged**